components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Gateway/src/Server/GatewayServer.cs

```diff
@@ -1600,14 +1602,16 @@ public sealed class GatewayServer : SessionServerBase, IServer {
                 ApplicationName = "Opc UA Gateway Server",
                 ApplicationType = Opc.Ua.ApplicationType.ClientAndServer,
                 ApplicationUri =
-                    $"urn:{Utils.GetHostName()}:Microsoft:OpcGatewayServer",
+                    $"urn:{Dns.GetHostName()}:Microsoft:OpcGatewayServer",
```

Comment:
>Dns.GetHostName() [](start = 27, length = 17)

the utils version strips potential fqdn, not sure if this could become an issue

Reply:
Utils and Dns Get HostByName were not consistently used through the implementation. I decide d to use Dns everywhere. Thought this could potentially be a reason of the automated tests sporadic failures.